### PR TITLE
Pull request for adding mobile menu in my portfolio

### PR DESCRIPTION
**Pull request by Hamayounali**
1. Updated style.css file. 
2. Updated index.html file. 
3. Created script.js file

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -878,3 +729,84 @@ body {
...
   }
 }
+
+/* Memu barr */
+.topnav {
+  overflow: hidden;
+  background-color: #333;
+  padding-left: 90px;
+  padding-right: 90px;
+}
```
Comment (vechicin) [inline assets/css/style.css]: Your design doesn't quite look like the template design.

Your design:

Screen Shot 2022-06-20 at 10 23 39

The template design:

Screen Shot 2022-06-20 at 10 24 15

Please fix it.

File: assets/css/style.css [hunk 2]
```diff
@@ -878,3 +729,84 @@ body {
...
+    right: 0;
+    top: 0;
+  }
+
+  .topnav.responsive a {
+    float: none;
+    display: block;
+    text-align: left;
+  }
+}
```
Comment (vechicin) [inline assets/css/style.css]: According to the principles of mobile-first development, media queries should only be used to make the design responsive for bigger screens. This media query is being applied for mobile screens. Please remove it.

Additionally, your design is not following the template design.

Your design:

Screen Shot 2022-06-20 at 10 26 44

Template design:

Screen Shot 2022-06-20 at 10 27 12

Please fix it.

Additionally, the mobile menu behavior is not the one expected. Please review what happens when the user clicks on the mobile menu links. Please fix the behavior.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Hamayounali ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good first attempt at implementing the mobile menu

## Required Changes ♻️

- [ ] You need to install ESLint since this project includes JavaScript. Please do so.

Screen Shot 2022-06-20 at 10 31 13

- [ ] According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#work-and-submission-mode) there needs to be contributions by both partners. I can only see one partner's contribution. If you are having issues with your coding partner, please reach out to Student Success and make a note in your review request about it.

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/css/style.css (hunk 2 above)
Comment (sja-thedude) [reply assets/css/style.css]: - [x] As suggested previously kindly remove media queries from the mobile version as it is only needed for the desktop.
- [x] Kindly make sure to follow the same mobile menu design as per the Figma design. Follow the same colour, spacing and font-weight for the same. Also, make sure to display the line after each menu option as per the design.

https://user-images.githubusercontent.com/70269507/174764938-04cd3c44-3e80-431a-9176-2264dfbe4e24.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Hamayounali, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Hamayounali** [comment]
Thank you Sayeda for the nice comments and review. I have removed the media query and updated the menu accordingly. I hope this time my project meets the requirements.

File: assets/css/style.css [hunk 3]
```diff
@@ -877,4 +728,92 @@ body {
...
+
+  .topnav.responsive {
+    position: relative;
+  }
+
+  .topnav.responsive .icon {
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
```
Comment (sja-thedude) [inline assets/css/style.css]: - [x] As suggested previously, kindly make changes as per the Figma design.
- [x] Your design has to be similar to the design on the left as per Figma.
- [x] The mobile menu should be of full length not half
- [x] The envelope icon should not display on the mobile menu and you should follow the same color text for menu options.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Hamayounali, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/css/style.css (hunk 3 above)
Comment (Hamayounali) [reply assets/css/style.css]: Thank you @sja-thedude for your time and nice review. I have updated the menu according to your comments.

File: index.html [hunk 4]
```diff
@@ -5,6 +5,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 	<title>Hamayoun</title>
 	<link rel="stylesheet" type="text/css" href="assets/css/style.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<script src="assets/js/script.js"></script>
 	<style>
 
 		@font-face {
```
Comment (rukundoeric) [inline index.html:12]: - [ ] <b>OPTIONAL:</b> It is not best practice to use internal CSS. It would nice to consider moving them to external file.

**Review by rukundoeric — APPROVED** [review]
Hi @Hamayounali,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] check the comments under review

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**